modules/support_ticket/support_ticket.module

```diff
@@ -266,6 +266,54 @@ function support_ticket_load($nid = NULL, $reset = FALSE) {
 }
 
 /**
+ * Determines what fields have changed when the support ticket is updated.
+ *
+ * @param object $support_ticket
+ *   The support ticket entity object.
+ */
+function support_ticket_entity_update($support_ticket) {
+  $original = $support_ticket->original;
+  $fields_original = $support_ticket->original->getFields();
+  $fields_updated = $support_ticket->getFields();
+
+  $comment = '';
+
+  // Don't log changes to the following fields when the support_ticket is updated.
+  // @todo: "field_ticket_update" is a configurable comment field; I think we can simply
+  //        ignore any comment field.
```

Comment:
What's the cleanest way to detect if a given field is a comment, and to ignore it here? Currently I assume we're still using the field that's pre-generated when the support_ticket module is installed.